code/modules/station_goals/vault_mutation.dm

```diff
@@ -0,0 +1,124 @@
+/datum/mutation/human/breathless
+	name = "Breathless"
+	desc = "A mutation within the skin that allows for filtering and absorption of oxygen from the skin."
+	text_gain_indication = "Your lungs feel great."
+	text_lose_indication = "Your lungs feel normal again."
+	locked = TRUE
+	mutadone_proof = TRUE
+
+/datum/mutation/human/breathless/on_acquiring(mob/living/carbon/human/acquirer)
+	. = ..()
+	ADD_TRAIT(acquirer, TRAIT_NOBREATH, DNA_VAULT_TRAIT)
+
+/datum/mutation/human/breathless/on_losing(mob/living/carbon/human/owner)//this shouldnt happen under normal condition but just to be sure
+	. = ..()
+	REMOVE_TRAIT(owner, TRAIT_NOBREATH, DNA_VAULT_TRAIT)
+
+/datum/mutation/human/fast_runner
+	name = "Fast runner"
+	desc = "A mution within the leg muscles that allows it to operate at 20% more than the usual capacity."
+	text_gain_indication = "Your legs feel faster and stronger."
+	text_lose_indication = "Your legs feel weaker and slower."
+	locked = TRUE
+	mutadone_proof = TRUE
+
+/datum/mutation/human/fast_runner/on_acquiring(mob/living/carbon/human/acquirer)
+	. = ..()
+	acquirer.add_movespeed_modifier(/datum/movespeed_modifier/dna_vault_speedup)
+
+/datum/mutation/human/fast_runner/on_losing(mob/living/carbon/human/owner)
+	. = ..()
+	owner.remove_movespeed_modifier(/datum/movespeed_modifier/dna_vault_speedup)
+
+/datum/mutation/human/tough_skin
+	name = "Tough skin"
+	desc = "A mutation within the epidermis that makes it more resistant to tear."
+	text_gain_indication = "Your skin feels tougher."
+	text_lose_indication = "Your skin feels weaker."
+	locked = TRUE
+	mutadone_proof = TRUE
+
+/datum/mutation/human/tough_skin/on_acquiring(mob/living/carbon/human/acquirer)
+	. = ..()
+	acquirer.armor += 30
+	ADD_TRAIT(acquirer, TRAIT_PIERCEIMMUNE, DNA_VAULT_TRAIT)
+
+/datum/mutation/human/tough_skin/on_losing(mob/living/carbon/human/owner)
+	. = ..()
+	owner.armor -= 30
+	REMOVE_TRAIT(owner, TRAIT_PIERCEIMMUNE, DNA_VAULT_TRAIT)
+
+/datum/mutation/human/dexterous
+	name = "Dexterous"
+	desc = "A mutation within the nerve system that allows for more responsive and quicker action."
+	text_gain_indication = "Your limbs feel more dexterous and responsive."
+	text_lose_indication = "Your limbs feel less dexterous and responsive."
+	locked = TRUE
+	mutadone_proof = TRUE
+
+/datum/mutation/human/dexterous/on_acquiring(mob/living/carbon/human/acquirer)
+	. = ..()
+	acquirer.next_move_modifier = 0.5
```

Comment:
you cannot directly set these. that overrides any other modifier the human could receive. use multipliers (same goes for other stuff)